config.py

```diff
@@ -81,13 +82,14 @@
 PACKAGE_MANAGER_COMMAND_INSTALL = None
 PACKAGE_MANAGER_COMMAND_QUERY = None
 PACKAGES = None
+L9PACKAGES = None
 SUPERUSER_COMMAND = None
 
 persistent_config_keys = [
     "FLPRODUCT", "FLPRODUCTi", "TARGETVERSION", "INSTALLDIR", "APPDIR",
     "APPDIR_BINDIR", "WINETRICKSBIN", "WINEPREFIX", "WINEBIN_CODE", "WINE_EXE",
-    "WINESERVER_EXE", "WINE64_APPIMAGE_FULL_URL", "WINECMD_ENCODING",
-    "WINE64_APPIMAGE_FULL_FILENAME", "APPIMAGE_LINK_SELECTION_NAME",
+    "WINESERVER_EXE", "APPIMAGE_FILENAME", "RECOMMENDED_WINE64_APPIMAGE_FULL_URL", "WINECMD_ENCODING",
+    "RECOMMENDED_WINE64_APPIMAGE_FULL_FILENAME", "APPIMAGE_LINK_SELECTION_NAME",
```

Comment:
Do we need to save `RECOMMENDED_WINE64_APPIMAGE` variables to the config file? It seems like they would be fine just being script/app global variables, right? It would also avoid any potential problems down the road of the recommended values change but the user's config keeps the old values. And I'm in favor of reducing the config file to only the fewest necessary variables.

Reply:
Yes—let's do that.. I'll also remove APPIMAGE_LINK_SELECTION_NAME, which can be a system level var.